projects/valtimo/dossier/src/lib/dossier-detail/tab/summary/summary.component.ts

```diff
@@ -24,15 +24,17 @@ import {
   ViewEncapsulation,
 } from '@angular/core';
 import {ProcessInstanceTask, ProcessService} from '@valtimo/process';
-import {ActivatedRoute, ParamMap, Router} from '@angular/router';
+import {ActivatedRoute, Router} from '@angular/router';
 import {Document, DocumentService, ProcessDocumentInstance} from '@valtimo/document';
 import {TaskDetailModalComponent, TaskService} from '@valtimo/task';
 import {FormService} from '@valtimo/form';
 import {FormioOptionsImpl, ValtimoFormioOptions} from '@valtimo/components';
 import moment from 'moment';
 import {FormioForm} from '@formio/angular';
+import {BehaviorSubject, combineLatest, defaultIfEmpty, forkJoin, map, Observable, of, Subscription, switchMap} from 'rxjs';
+import {SseService, TaskUpdateSseEvent} from '@valtimo/sse';
```

Comment:
We don't want to make this dependency mandatory yet: `@valtimo/sse`

Reply:
We decided to not use the `@valtimo/sse` in release `10.5.0` yet, due several issues.